Spec/API.md

```diff
@@ -355,7 +355,8 @@ This response only contains metadata (does not include the cards in the deck).
     "created": "<created>"
```

Comment:
GitHub won't let me select this, but should `user_data-version` be `user_data_version`?